spark/src/main/scala/org/apache/iceberg/spark/SparkTableUtil.scala

```diff
@@ -297,5 +302,81 @@ object SparkTableUtil {
       )
     }
   }
+
+  private def buildManifest(table: Table,
+      sparkDataFiles: Seq[SparkDataFile],
+      partitionSpec: PartitionSpec): ManifestFile = {
+    val outputFile = table.io
+      .newOutputFile(FileFormat.AVRO.addExtension("/tmp/" + UUID.randomUUID.toString))
```

Comment:
Agreed. Just want to confirm why it can't be the local file? It is just a temporary file, I wanted to store it in the tmpfs of localhost to save some IO.

Reply:
Understood, I was still thinking serialize the manifest file...